### DISCUSSION TITLE
这么多版本选择哪个啊

### DISCUSSION THREAD

**Discussion by nikefu** [2dust/v2rayNG#3594] [Q&A]
![Screenshot_2024-09-24-17-51-46-606_com.github.android.jpg](https://github.com/user-attachments/assets/96af75af-caf9-4271-adc3-ec784da05b3a)

每个版本都一样吗 安卓应该怎么选呢

**Comment by nikefu** [comment]
v2rayNG_1.8.39_arm64-v8a.apk
23 MB
v2rayNG_1.8.39_armeabi-v7a.apk
23 MB
v2rayNG_1.8.39_universal.apk
60MB
v2rayNG_1.8.39_x86.apk
25 MB
v2rayNG_1.8.39_x86_64.apk
24 MB
就是这些，不知道应该下哪一个

**Reply by 5lonely** [reply]
不知道就选第三个

**Reply by gydchenxiao** [reply]
> v2rayNG_1.8.39_arm64-v8a.apk 23 MB v2rayNG_1.8.39_armeabi-v7a.apk 23 MB v2rayNG_1.8.39_universal.apk 60MB v2rayNG_1.8.39_x86.apk 25 MB v2rayNG_1.8.39_x86_64.apk 24 MB 就是这些，不知道应该下哪一个

现在正常安卓手机选第一个吧

**Reply by 13098724567** [reply]
安卓手机就选arm64后缀是apk的那个
